zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -70,7 +70,8 @@ struct RemoteApplicationResult {
  */
```

Comment:
Actually I wrote this description to address [this feedback](https://github.com/apache/zeppelin/pull/984#discussion_r66837016) by @bzz. So why don't you add some description about what `meta` variable do for the future contributors? :)